scrapy_zyte_smartproxy/__init__.py

```diff
@@ -1,4 +1 @@
-from .middleware import ZyteSmartProxyMiddleware
```

Comment:
This is not good.

The “unused import” thing should generally be ignored in the main `__init__.py` of a package, where we usually put imports to make it easier for people to import them, e.g. `from scrapy_zyte_smartproxy import ZyteSmartProxyMiddleware` instead of `from scrapy_zyte_smartproxy.middleware import ZyteSmartProxyMiddleware`.